src/graphql-client.ts

```diff
@@ -20,21 +18,9 @@ const headers = {
 
 const uri = "https://api.github.com/graphql";
 
-const fragmentMatcher = new IntrospectionFragmentMatcher({
-    introspectionQueryResultData: {
-        __schema: {
-            types: []

```

Comment:
FragmentMatcher, HeuristicFragmentMatcher, and IntrospectionFragmentMatcher have all been removed https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#breaking-cache-changes:~:text=FragmentMatcher%2C%20HeuristicFragmentMatcher%2C%20and%20IntrospectionFragmentMatcher%20have%20all%20been%20removed

`types` was empty so we weren't actually using it anyway.